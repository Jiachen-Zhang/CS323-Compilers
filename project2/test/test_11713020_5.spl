struct Apple
{
  int weight;
};
int test_11713020_5()
{
  struct Apple aa;
  aa.weight = b;
}