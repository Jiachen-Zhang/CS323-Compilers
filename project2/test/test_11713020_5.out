Error type 1 at Line 8: undefined variable: b
Error type 5 at Line 8: unmatching type on both sides of assignment
